sdk/purview/azure-analytics-purview-sharing/src/test/java/com/azure/analytics/purview/sharing/PurviewShareTestBase.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
nit: to remove this header in the files where you wrote the code rather than generated else people may try to find where to modify the 'source' when editing tests.

Reply:
Thanks this should be addressed in the latest push.